run.py

```diff
@@ -872,6 +872,7 @@ def run_test(dirpath, args, cwd, suricata_config):
         elif results["success"] > 0:
             with lock:
                 count_dict["passed"] += 1
+        subprocess.call(["../suricata-verify/check-eve.py",outdir])
```

Comment:
Since we run the script after each test, it prints the pass fail count after each count.

Reply:
This doesn't seem to be working for me.  I updated the schema so all checks would fail, but it will passes.

A few ideas:
- We probably have to check the return value of this function.  If it fails, we'd throw a TestError with a reason which would be caught in the except block.
- We probably want to prevent any output when calling `check-eve.py` here.  A common command like option is `-q` and `--quiet` to suppress all output.  Then `check-eve` would exit with an error code..  It should exit with an error code if any checked json files fail anyways.